graphql-cohttp/test/dune

```diff
@@ -0,0 +1,4 @@
+(tests
+  (package graphql-cohttp)
+  (names test)
+  (libraries cohttp-lwt-unix digestif.ocaml graphql-lwt graphql-cohttp alcotest))
```

Comment:
I think this makes it incompatible with Mirage?

Reply:
ooops, meant to comment on the `.opam` file diff...

Reply:
These are just test dependencies, so I believe it should be fine?